pkgs/top-level/all-packages.nix

```diff
@@ -6312,6 +6312,8 @@ in
 
   redo = callPackage ../development/tools/build-managers/redo { };
 
+  reno = pythonPackages.reno;
```

Comment:
yep, unfortunately apps still end up in there. But the docs state what should be where.